app/models/main_page_refresher.rb

```diff
@@ -0,0 +1,28 @@
+module MainPageRefresher
```

Comment:
@dissident looks like it's concern and should be put in `app/models/concern` http://api.rubyonrails.org/classes/ActiveSupport/Concern.html